src/Data/BitVector/Sized/Signed.hs

```diff
@@ -126,3 +128,11 @@ instance (KnownNat w, 1 <= w) => Ix (SignedBV w) where
 instance (KnownNat w, 1 <= w) => Bounded (SignedBV w) where
   minBound = SignedBV (BV.minSigned knownNat)
   maxBound = SignedBV (BV.maxSigned knownNat)
+
+instance KnownNat w => UniformRange (SignedBV w) where
+  uniformRM (SignedBV lo, SignedBV hi) g = SignedBV <$> uniformRM (lo, hi) g
```

Comment:
These are stored in "unsigned" value, right?  You probably need to convert to the signed value to make this range make sense.

Reply:
Ah yes, of course. Which makes me think... perhaps we should only include these instances for `UnsignedBV` and `SignedBV`.